Project/Source/Modules/ModuleWindow.cpp

```diff
@@ -129,6 +139,47 @@ void ModuleWindow::SetTitle(const char* title) {
 	SDL_SetWindowTitle(window, title);
 }
 
+void ModuleWindow::SetCursor(bool isPlaying) {
+	if (isPlaying) {
+		// Load resource
+		Scene* scene = App->scene->scene;
+		ResourceTexture* cursorResourceTexture = App->resources->GetResource<ResourceTexture>(scene->GetCursor());
+		if (cursorResourceTexture == nullptr) {
+			LOG("Error cursorResourceTexture null");
+			return;
+		}
```

Comment:
I would not include this LOG here.
Not having a cursor is an option, not an error!

Remember to merge develop into the branch, and test the feature in GAME configuration